.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+  schedule:
+  - cron: '44 13 * * *'
```

Comment:
We should comment that this was arbitrarily chosen

Reply:
Done!